doc/whats_new/v0.21.rst

```diff
@@ -542,6 +542,12 @@ Multiple modules
 - |Efficiency| Memory copies are avoided when casting arrays to a different
   dtype in multiple estimators. :issue:`11973` by :user:`Roman Yurchak
   <rth>`.
+- Add estimators tags: these are annotations of estimators that allow
+  programmatic inspection of their capabilities, such as sparse matrix
+  support, supported output types and supported methods. The estimators checks
+  are now run selectively, depending on the estimator's tags. Read more in the
```

Comment:
This may sounds a bit confusing, as estimator checks were always run selectively. Maybe,
```
Estimator tags also determine tests that are on an estimator when `check_estimator` is called.
```